### DISCUSSION TITLE
No connection to Redis in Docker

### DISCUSSION THREAD

**Discussion by HallerPatrick** [outline/outline#4874] [Self Hosting]
I am getting the error that no connection with redis can be established on a MacBook and a remote ubuntu server.

I am following the instructions for the setup with docker https://wiki.generaloutline.com/s/hosting/doc/docker-7pfeLP5a8t
I assume that the default configuration for redis should be sufficient and therefore work out of the box. Because I am getting the same error in two different setup, I am not sure this follows the expected behaviour.

**To Reproduce**
Follow the instructions as shown in https://wiki.generaloutline.com/s/hosting/doc/docker-7pfeLP5a8t

**Expected behavior**
`outline_1` can communicate with redis service

**Screenshots**
```
outline_1       | {"level":"warn","message":"Retrying redis connection: attempt 1"}
outline_1       | {"level":"warn","message":"Retrying redis connection: attempt 2"}
outline_1       | {"level":"warn","message":"Retrying redis connection: attempt 3"}
outline_1       | {"level":"warn","message":"Retrying redis connection: attempt 4"}
```

**Outline (please complete the following information):**
- Install: self hosted
- Version: 60101c507a243afb9982adab2cf7d90b78daff22

**Comment by vgwidt** [comment]
What do the logs show on the redis container?

**Comment by dwarsbit** [comment]
For me the default didn't work either. I got it working by setting the `REDIS_URL` to `redis://redis:6379`

**Reply by AutonomousWork** [reply]
Thanks for posting this! Ran into the issue earlier today and this was the fix
